canonical-web-and-design/ubuntu.com
# 19.10 homepage and engage page

## Done

- 19.10 homepage and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
    - http://0.0.0.0:8001/
    - http://0.0.0.0:8001/takeovers
    - http://0.0.0.0:8001/engage/19-10-webinar
    - http://0.0.0.0:8001/engage/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5852) and the [brief](https://docs.google.com/spreadsheets/d/1SLmQWXO4P3iJdka4P0N1A95d755cLRIt68sxdocK7jw/edit#gid=0)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1820
